plugins/Files/js/fileElement.js

```diff
@@ -47,14 +77,22 @@ function updateFileElement(f, el) {
 
 // Make file element with jquery
 function makeFileElement(f) {
+	// Get extension from name
+	var re = /(?:\.([^.]+))?$/;
+	var ext = re.exec(f.name)[1]; 
```

Comment:
Alternatively, you could use: https://nodejs.org/api/path.html#path_path_extname_p